src/teleop/download_map/download_map.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import zipfile
+import json
+import boto3
+import subprocess
+
+
+# If System has issues with number of file watcher when using GUI with downloaded map
+# Run "sudo sysctl -w fs.inotify.max_user_watches=100000"
+def main():
+    result = subprocess.Popen(
+        ["rospack", "find", "mrover"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )  # runs rospack find mrover to get directory
+    m_bytes, err = result.communicate()
+    m = m_bytes.decode("utf-8")  # convert byte to string type
+    m = m[: len(m) - 1]  # the old m includes the '\n' and this gets rid of that
```

Comment:
You should just be able to say `m[:-1]`

Reply:
This is very common practice in python so probably even more readable

Reply:
Depending on how you feel, you could also just include this slicing into the previous line